packages/k6-tests/scripts/localhost.js

```diff
@@ -0,0 +1,17 @@
+import { check, group, sleep } from 'k6';
+import http from 'k6/http';
+import exec from 'k6/execution';
+
+function log() {
+  console.log('Env Vars -->');
+  console.log(`  K6_SCRIPT_SDK_ENDPOINT_URL=${__ENV.K6_SCRIPT_SDK_ENDPOINT_URL},`);
+}
+
+export function localhost() {
+  !exec.instance.iterationsCompleted && (exec.vu.idInTest === 1) && log();
+  group('Local test', function () {
+    let res = http.get(__ENV.K6_SCRIPT_SDK_ENDPOINT_URL || 'http://127.0.0.1:8080');
```

Comment:
Do you want to reuse this variable? Or is it a mistake? K6_SCRIPT_SDK_ENDPOINT_URL